WordPress/src/main/java/org/wordpress/android/ui/photopicker/PhotoThumbnailViewHolder.kt

```diff
@@ -2,16 +2,25 @@ package org.wordpress.android.ui.photopicker
 
 import android.view.ViewGroup
 import android.widget.ImageView
+import android.widget.ImageView.ScaleType.FIT_CENTER
 import android.widget.TextView
 import org.wordpress.android.R
+import org.wordpress.android.util.image.ImageManager
+import org.wordpress.android.util.image.ImageType.PHOTO
 
 /*
  * ViewHolder containing a device thumbnail
  */
 
-@Deprecated("This class is being refactored, if you implement any change, please also update " +
-        "{@link org.wordpress.android.ui.mediapicker.PhotoThumbnailViewHolder}")
-class PhotoThumbnailViewHolder(parent: ViewGroup, private val thumbnailViewUtils: ThumbnailViewUtils) :
+@Deprecated(
+        "This class is being refactored, if you implement any change, please also update " +
+                "{@link org.wordpress.android.ui.mediapicker.PhotoThumbnailViewHolder}"
+)
+class PhotoThumbnailViewHolder(
+    parent: ViewGroup,
+    private val thumbnailViewUtils: ThumbnailViewUtils,
+    private val imageManager: ImageManager
+) :
```

Comment:
🔍 Nitpick: The linebreak seems unnecessary.

Reply:
Good catch, I've fixed the other linebreaks as well in e30225ad41